src/electionguard_gui/services/export_service.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+
+def get_drives() -> list[str]:
+    dl = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    drives = [f"{d}:\\" for d in dl if os.path.exists(f"{d}:")]
```

Comment:
Do we want to list all drives or just the removable ones?

Reply:
considering that the devices only have 1 fixed drive and then thumb drives, this should be ok for now.  Thought for the future

Reply:
agreed, that will be a nice enhancement for the future